packages/app/pages/index.tsx

```diff
@@ -7,7 +7,7 @@ export default function Home() {
   const profile: User = useProfile();
 
   if (profile) {
-    Router.push("/today");
+    Router.push("/class/" + profile.courses[0].course.id + "/today");
```

Comment:
the rule for `next/link` applies for Router.push as well:
https://nextjs.org/docs/api-reference/next/router#usage
see
```
Router.push('/post/[pid]', '/post/abc')
```
first argument is the file system path to the page and should be static.